Batch8CucumberSeleniumAPI/src/test/resources/features/ui/Search.feature

```diff
@@ -15,12 +15,22 @@ Feature: Search functionality
 	  And I click on hamburger menu with category as "Mobiles, Computers"
 	  And I click on hamburger menu with sub category as "Power Banks"
 	  Then Search results are displayed for products related to "Power Banks"
-	  
-	  
-	 @TC0008 
+	
+	@TC0008 
 	 Scenario: A User Should Be Able To Click On A Product Link From The Search Page And Product Description Page Should Be Displayed.
 		Given I have browser opened and url is navigated
 		When I search for product as "Dell"
 		And I click on any product in the Search Result
 		Then I am able to see product description and detail in new tab
-	  
\ No newline at end of file
+	
+	@TC0033  
+	Scenario: A user should be able to search a product under price range
```

Comment:
There is difference in your implementation of the test case please read it again sharing the link of QA Touch TC0033
https://visionit.qatouch.com/case/view/p/EEay/cid/784y1

Reply:
Check it and change the steps

Reply:
Ok. Thank you sarang for your review

Reply:
I'll do it.